src/Grid/topology.jl

```diff
@@ -1,4 +1,4 @@
-
+include("../ArrayOfVectorViews.jl")
```

Comment:
This should go into `src/Ferrite.jl` right?

Reply:
Yes, left it temporarily here but should probably move it. Will do when putting into a module!